tests/test_rdf/vegetables.rdf

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:skos="http://www.w3.org/2004/02/skos/core#">
+    <skos:ConceptScheme rdf:about="http://example.com/vegetables">
+        <skos:prefLabel xml:lang="en">Vegetables</skos:prefLabel>
+    </skos:ConceptScheme>
+    <skos:Concept rdf:about="http://example.com/vegetables/potato">
+        <skos:inScheme rdf:resource="http://example.com/vegetables"/>
+        <skos:prefLabel xml:lang="en">Potato</skos:prefLabel>
+    </skos:Concept>
+</rdf:RDF>
```

Comment:
The kernel tests are very difficult to read and understand, can we improve them using a YML file to have the different scenario, as we are using in other projects?

Reply:
Not sure why this comment goes on the RDF file...

But in any case, the Kernel tests are basic entity API testing of the entities: load, query, data values. There is nothing overtly complicated there.